docs/sources/enterprise/license-restrictions.md

```diff
@@ -25,6 +25,12 @@ Restrictions are applied separately for viewers and editors.
 
 When the number of maximum active viewers or editors is reached, Grafana displays a warning banner.
 
+### Usage billing
+
+It is possible to turn on usage billing in Enterprise licensing in order to have an unlimited number of user limits. When usage billing is on, Grafana does not enforce active user limits. Instead, Grafana charges for active users above the limit, according to your customer contract.
```

Comment:
```suggestion
It is possible to turn on usage billing in Enterprise licensing in order to have an unlimited number of user limits. When usage billing is on, Grafana does not enforce active user limits. Instead, Grafana Labs charges for active users above the limit, according to your customer contract.
```

@osg-grafana I'm not sure about this, but I think we might want to refer to the billing entity as "Grafana Labs" (or something like "your Grafana vendor", but I don't think that's common enough that we need to care about that at the moment)?